types/jwt-decode/jwt-decode-tests.ts

```diff
@@ -1,7 +1,6 @@
- 
-import jwtDecode = require('jwt-decode');
- 
-let token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJmb28iOiJiYXIiLCJleHAiOjEzOTMyODY4OTMsImlhdCI6MTM5MzI2ODg5M30.4-iaDojEVl0pJQMjrbM1EzUIfAZgsbK_kgnVyVxFSVo";
+import * as jwtDecode from "jwt-decode";
```

Comment:
The old import statement was correct as ```jwt-decode``` isn't an ES6 module ([explanation](https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this))
```ts 
import jwtDecode = require('jwt-decode')
```